### PR TITLE
Wish list(add style to buttons and for deletecontainer)

### PR DESCRIPTION
**Pull request by VitalikVoicix**

### CODE REVIEW

**Review by DmytroDidukh — APPROVED** [review]

**Review by Iryna-Bzdyr — APPROVED** [review]

File: src/components/modal/modal.styles.js [hunk 1]
```diff
@@ -23,6 +27,7 @@ export const useStyles = makeStyles((theme) => ({
       backgroundColor: theme.palette.button.normal.backgroundColor,
       padding: 5,
       margin: 5,
+      width: '150px',
```
[the remark this answers is not in the document]
Comment (VitalikVoicix) [reply src/components/modal/modal.styles.js]: Allready change and made responsive design

File: src/components/modal/modal.styles.js [hunk 2]
```diff
@@ -1,32 +1,84 @@
...
+      paddingBottom: '5px',
+      margin: 3,
+      width: '130px',
       '&:hover': {
         color: theme.palette.button.hover.color,
         backgroundColor: theme.palette.button.hover.backgroundColor
       }
     }
   }
+  // '@media (max-width: 950px)': {
```
Comment (moran711) [inline src/components/modal/modal.styles.js:62]: Why you commented this lines?

File: src/components/modal/modal.styles.js [hunk 3]
```diff
@@ -1,32 +1,84 @@
...
+    color: 'black',
+    border: '5px solid #666',
     boxShadow: theme.shadows[5],
-    padding: theme.spacing(2, 4, 3),
-    '& p': {
-      fontSize: '1.2em'
-    }
+    padding: theme.spacing(2, 4, 3)
+
+    //   '@media (max-width: 950px)': {
```
Comment (moran711) [inline src/components/modal/modal.styles.js:36]: Why you commented this lines?

File: src/components/modal/modal.styles.js [hunk 4]
```diff
@@ -1,32 +1,84 @@
...
+  //     float: 'left'
+  //   }
+  // },
   paper: {
     position: 'absolute',
-    top: '50%',
-    left: '50%',
+    top: '53%',
+    left: '51%',
+    // marginLeft:'50%',
```
Comment (moran711) [inline src/components/modal/modal.styles.js:25]: Why you commented this lines?

Comment (VitalikVoicix) [reply src/components/modal/modal.styles.js:25]: I guess I have just forgot to delete this commented code.Now it is deleted

File: src/components/modal/modal.styles.js (hunk 2 above)
Comment (VitalikVoicix) [reply src/components/modal/modal.styles.js:62]: I guess I have just forgot to delete this commented code.Now it is deleted

File: src/components/modal/modal.styles.js (hunk 3 above)
Comment (VitalikVoicix) [reply src/components/modal/modal.styles.js:36]: I guess I have just forgot to delete this commented code.Now it is deleted

**Review by moran711 — APPROVED** [review]

**Pull request merged**